src/domain/createBuildRoute/index.js

```diff
@@ -42,10 +42,10 @@ function getParams(args) {
   return undefined;
 }
 
-function normalizeUrl(url) {
-  let result = url.replace(/[^:]\/\//g, '/'); // Gets rid of duplicated slashes (//)
+export function normalizeUrl(url) {
+  let result = url.replace(/(^|[a-z0-9])\/+/gi, '$1/'); // Gets rid of duplicated slashes (//)
```

Comment:
Being picky but `"base_//derp"` still won't be processed even though it's a valid path. How about `/(^|[\w-])\/+/gi`?